.docker/indexd.conf

```diff
@@ -0,0 +1,19 @@
+<VirtualHost *:80>
+  LogLevel warn
+  WSGIProcessGroup indexd
+  WSGIApplicationGroup %{GLOBAL}
+  WSGIScriptReloading On
+  WSGIPassAuthorization On
+  WSGIDaemonProcess indexd processes=4 threads=8 home=/var/www/indexd/ python-path=/venv/lib/python3.8/site-packages/:/var/www/indexd
```

Comment:
Not related to this PR but given that Python 3.8 is no longer supported, should we remove support from indexd?

Reply:
We should, but outside the scope of this work as you mentioned. The hardcoded python path is also one more place we need to remember to update(in tungsten) after upgrading python version.